attractions/file-input/file-dropzone.svelte

```diff
@@ -26,20 +27,27 @@
     wrongType = [...e.dataTransfer.items].some(file => !accepted(accept, file));
   }
 
-  function acceptUpload(e) {
+  async function acceptUpload(e) {
     for (let file of (e.dataTransfer || e.target).files) {
```

Comment:
Consider using `.forEach` so that it doesn't wait for `beforeChange` to resolve (or reject) before moving on to the next file